testsuite/forge-cli/src/main.rs

```diff
@@ -542,14 +549,36 @@ fn single_test_suite(test_name: &str) -> Result<ForgeConfig<'static>> {
                 None,
                 None,
             )),
+        // TODO: Add tracing latency of high-gas-fee transactions
+        "graceful_overload" => config
+            .with_initial_validator_count(NonZeroUsize::new(10).unwrap())
+            .with_initial_fullnode_count(4)
+            .with_network_tests(vec![&PerformanceBenchmarkWithFN])
+            .with_emit_job(EmitJobRequest::default().mode(EmitJobMode::ConstTps { tps: 15000 }))
+            .with_genesis_helm_config_fn(Arc::new(|helm_values| {
+                // Have single epoch change in land blocking
+                helm_values["chain"]["epoch_duration_secs"] = 300.into();
+            }))
+            .with_success_criteria(SuccessCriteria::new(
+                5500,
+                50000,
+                true,
+                Some(Duration::from_secs(120)),
+                Some(SystemMetricsThreshold::new(
+                    // Check that we don't use more than 12 CPU cores for 30% of the time.
+                    MetricsThreshold::new(12, 30),
+                    // Check that we don't use more than 5 GB of memory for 30% of the time.
+                    MetricsThreshold::new(15 * 1024 * 1024 * 1024, 30),
```

Comment:
Any reason we need so high memory for this? Is this because of mempool filling up?

Reply:
wanted to run and see what the value actually is, will update after a run, just noticed it didn't run once will check why